vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/RawWebSocketTransport.java

```diff
@@ -144,6 +142,15 @@ public SockJSSocket endHandler(Handler<Void> endHandler) {
       return this;
     }
 
+    @Override
+    public SockJSSocket closeHandler(Handler<Void> closeHandler) {
+      ws.closeHandler(v -> {
+        defaultCloseHandler.handle(v);
+        closeHandler.handle(v);
```

Comment:
Shouldn't we check if the closeHandler is set? As I see above there is a setter, so if a user sets `null` this will NPE I guess.